web/src/stream_settings_ui.js

```diff
@@ -440,8 +442,25 @@ export function render_left_panel_superset() {
     ui.get_content_element($("#manage_streams_container .streams-list")).html(html);
 }
 
+export function render_empty_left_panel_message() {
+    const is_empty = $("#manage_streams_container .stream-row:not(.notdisplayed)").length === 0;
+    if (!is_empty) {
+        $(".nothing-to-show").hide();
+        return;
+    }
+    if (is_subscribed_stream_tab_active()) {
+        $(".create_stream_link_container").hide();
+        $(".view_all_stream_link_container").show();
+    } else {
+        $(".view_all_stream_link_container").hide();
+        $(".create_stream_link_container").show();
+    }
+    $(".nothing-to-show").show();
+}
+
 // LeftPanelParams { input: String, subscribed_only: Boolean, sort_order: String }
 export function redraw_left_panel(left_panel_params = get_left_panel_params()) {
+    $(".nothing-to-show").hide();
```

Comment:
I am not sure why is this added. Is this to ensure that the text is not shown when loading the streams or something else?

Reply:
Yes, Initially the message is not hidden so after loading the streams it hides the message.

Reply:
Yes, Initially the message is not hidden so after loading the streams it hides the message.

Reply:
Sorry, Earlier it was there and the reason is solved in function but it went un-noticed, removed it in latest commit.
Thank you !!